matrix_sdk_base/src/media.rs

```diff
@@ -0,0 +1,83 @@
+//! Common types for [media content](https://matrix.org/docs/spec/client_server/r0.6.1#id66).
+
+use std::fmt;
+
+use matrix_sdk_common::{
+    api::r0::media::get_content_thumbnail::Method, events::room::EncryptedFile,
+    identifiers::MxcUri, UInt,
+};
+
+/// The requested format of a media file.
+#[derive(Clone, Debug)]
+pub enum MediaFormat {
+    /// The file that was uploaded.
+    File,
+
+    /// A thumbnail of the file that was uploaded.
+    Thumbnail(MediaThumbnailSize),
+}
+
+impl fmt::Display for MediaFormat {
```

Comment:
`Display` isn't the correct trait here, since it generally creates a user presentable string. I think a dedicated method to create the key makes more sense. (same for other structs in this file)

Reply:
Yeah I know I was being lazy :P.

I thought about using `EncodeKey` but it's declared only in the sled store.

Reply:
i don't think we need to use the same key for memory store and sled store.

Reply:
We don't _need_ it, but we could. Seeing as it is basically the same format.

Reply:
sure, probably makes sense.